corehq/apps/app_manager/static/app_manager/js/report-module.js

```diff
@@ -287,18 +298,19 @@ var ReportModule = (function () {
                     data: {
                         name: JSON.stringify(self.moduleName),
                         module_filter: self.moduleFilter,
-                        reports: JSON.stringify(_.map(self.reports(), function (r) { return r.toJSON(); }))
+                        reports: JSON.stringify(_.map(self.reports(), function (r) { return r.toJSON(); })),
+                        multimediaUpdate: JSON.stringify(self.multimediaUpdate())
```

Comment:
picky, but I'd just call this `multimedia`, everything in this hash is an update